index.js

```diff
@@ -334,11 +324,25 @@ AFRAME.registerComponent('teleport-controls', {
 
     collisionEntities = [].slice.call(el.sceneEl.querySelectorAll(data.collisionEntities));
     this.collisionEntities = collisionEntities;
+    if (!this.data.collisionEntities) {
+      meshes = this.defaultCollisionMeshes;
+    } else {
+      meshes = this.collisionEntities.map(function (entity) {
+        return entity.getObject3D('mesh');
+      }).filter(function (n) { return n; });
+      meshes = meshes.length ? meshes : this.defaultCollisionMeshes;
+    }
```

Comment:
This gets repeated 3 times:
```
meshes = this.collisionEntities.map(function (entity) {
        return entity.getObject3D('mesh');
      }).filter(function (n) { return n; });
      meshes = meshes.length ? meshes : this.defaultCollisionMeshes;
```
Perhaps it's best to pull it out into a function?